### DISCUSSION TITLE
How to change the editor's font?

### DISCUSSION THREAD

**Discussion by ghost** [fredvs/ideU#59] [Q&A] [answered]
I wanted to change it to something else but I can't find where the setting is.

**Comment by fredvs** [comment, marked as answer]
The font of editor is defined by the project options:

![project-ideu](https://user-images.githubusercontent.com/3421249/204161934-29c5f6a9-5699-46a4-a809-b0210c08e368.png)

If you prefer to have your own config for all projects, independent of the options of the project, go to menu / _Settings_ / _Extra settings_ and, in right side of window, select in _Source Editor_: "_X Use those options instead of project options_".
And set all the values that follow like you want.

![set-editor-ideu](https://user-images.githubusercontent.com/3421249/204315010-2b3f4477-ae93-4550-b5e6-2ff38dd6944f.png)
